docs/drafts/version-ranges.md

```diff
@@ -0,0 +1,49 @@
+# Operator version ranges
+
+This document explains how to specify a version range to install or update an Operator with OLM 1.0.
+
+You define an Operator's target version in its custom resource (CR) file.
+The following list describes how OLM resolves an Operator's target version, and the resulting actions:
+
+Not specifying a version in the CR
+: Installs or updates the latest version of the Operator.
+Updates are applied automatically when they are published to the catalog.
+
+Specifying a version in the CR
+: Installs or updates the specified version.
+Updates are not applied automatically.
+If you want to update the Operator, you must manually edit the CR and apply the changes to the cluster.
+
+Specifying a channel
+: Installs or updates the latest version of the Operator in the channel.
+Updates are applied automatically when they are published to the specified channel.
```

Comment:
I would expect this doc to be purely about `spec.version`, so I'd suggest striking this.

In a higher-level doc, I'd expect something that talks about the various `spec` fields and their interactions.

Reply:
Removed.